bin/generate_resource_backend

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker-compose run --rm -w /app/src/backend/marsha app cookiecutter .cookiecutter/ "$@"
```

Comment:
I will not name it `resource` but `app` WDYT ? This is what you are really generating

Reply:
In fact, the purpose of this is to generate an LTI resource, which is an app indeed.
But I find it more descriptive like this, and the generated app embeds all LTI needs.

Reply:
name it `lti_app` in that case, `resource` is wrong IMO. `generate_backend_lti_app` is more accurate IMO.